styles.css

```diff
@@ -0,0 +1,266 @@
+* {
+  box-sizing: border-box;
+}
+
+body {
+  margin: 0;
+  height: 100vh;
+}
+body .content {
+  isolation: isolate;
+  height: 100%;
+  display: flex;
+  flex-direction: column;
+}
+body .content .desktop {
+  display: flex;
+  /* flex-direction: column; */
+  flex-direction: row;
+  gap: 1rem;
+}
+body .content .desktop .folder {
+  display: flex;
+  flex-wrap: wrap;
+}
+body .content .desktop button {
+  margin: 2rem;
+  border: none;
+  height: 6rem;
+  width: 6rem;
+  background: none;
+}
+
+body .content .desktop button img {
+  height: 5rem;
+  width: 5rem;
+  border-radius: 5pm;
+
+}
+body .content .desktop .form {
```

Comment:
change to scss file, open modal on top of icons, you can look into z-index: `https://www.w3schools.com/howto/tryit.asp?filename=tryhow_css_modal`